integrations-and-supported-tools/airflow/scripts/run_examples.sh

```diff
@@ -0,0 +1,13 @@
+set -e
+
+echo "Installing Airflow with constraints..."
+pip install "apache-airflow==2.6.3" --constraint "https://raw.githubusercontent.com/apache/airflow/constraints-2.6.3/constraints-3.7.txt"
```

Comment:
It is recommended to have constraints file: https://airflow.apache.org/docs/apache-airflow/stable/installation/installing-from-pypi.html#why-we-need-constraints